hexkit/protocols/eventpub.py

```diff
@@ -38,4 +37,20 @@ def publish(
             topic (str): The event type. ASCII characters only.
         """
         check_ascii(type_, key, topic)
+        await self._publish_validated(
```

Comment:
This separation makes sense, let's do it.

Reply:
Great, thanks.